games-strategy/openxcom/openxcom-1.0.recipe

```diff
@@ -0,0 +1,67 @@
+SUMMARY="Open-source clone of the original X-Com"
+DESCRIPTION="OpenXcom is an open-source clone of the original UFO: Enemy \
+Unknown (X-Com: UFO Defense in USA), licensed under the GPL and written in \
+C++ / SDL. It was originally founded by SupSuper in February 2010, and has \
+since grown into a small development team surrounded by a very supporting \
+community."
+HOMEPAGE="http://openxcom.org/"
+COPYRIGHT="2010-2014 OpenXcom Developers"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://codeload.github.com/SupSuper/OpenXcom/tar.gz/v$portVersion"
+CHECKSUM_SHA256="45acb280010a01d60506b1c5f2951ae501c012cc6161aac470bd15c1e6981246"
+SOURCE_DIR="OpenXcom-$portVersion"
+PATCHES="openxcom-$portVersion.patchset"
+ADDITIONAL_FILES="openxcom.rdef"
+
+ARCHITECTURES="!x86_gcc2 ?x86_64 ?x86"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	openxcom = $portVersion
+	app:OpenXcom
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libgl$secondaryArchSuffix
```

Comment:
I think the dependency on OpenGL is brought in by SDL, in that case this is fine.